libopenage/console/console.cpp

```diff
@@ -60,6 +63,9 @@ void Console::register_to_engine(Engine *engine) {
 	engine->register_drawhud_action(this);
 	engine->register_resize_action(this);
 
+	// create & register the TailSink
+	tailsink = new log::TailSink();
```

Comment:
to simplify this, you can make the class member a `std::unique_ptr<log::TailSink>` and initialize it with `this->tailsink = std::make_unique<log::TailSink>();`
Then you don't need the delete in the `~`.